src/components/BaseLayerSwitcher/BaseLayerSwitcher.scss

```diff
@@ -0,0 +1,58 @@
+.rs-base-layer-switcher {
+  position: relative;
+  height: 80px;
+  width: 115px;
+  transition: 250ms width;
+  overflow: hidden;
+  display: flex;
+
+  &.rs-open {
+    width: 600px;
+  }
+
+  .rs-base-layer-switcher-button {
+    margin: 4px;
+    background-color: white;
+    border: 2px solid white;
```

Comment:
maybe using display flex here, will fix the issue i mentionned further.

Reply:
forget I've seen there is an opacity on  the title.

Reply:
This has been fixed using flex, since the image is inserted as background now